packages/scandipwa/src/component/FieldNumber/FieldNumber.component.js

```diff
@@ -56,7 +56,7 @@ export class FieldNumber extends PureComponent {
                   type={ FIELD_TYPE.number }
                   readOnly
                   aria-label={ __('Value') }
-                  value={ value }
+                  value={ attr.value || value }
```

Comment:
Since this is the same value as on line 46, you could reuse the `numberValue` here.